syft/create_sbom.go

```diff
@@ -20,7 +21,10 @@ import (
 	"github.com/anchore/syft/syft/source"
 )
 
-func CreateSBOM(src source.Source, cfg CreateSBOMConfig) (*sbom.SBOM, error) {
+func CreateSBOM(ctx context.Context, src source.Source, cfg *CreateSBOMConfig) (*sbom.SBOM, error) {
```

Comment:
missing docstring, should at least contain hint for `nil` config for default behavior